app/models/relationship.rb

```diff
@@ -1,2 +1,6 @@
 class Relationship < ApplicationRecord
+    belongs_to :follower, class_name: "User"
```

Comment:
Fix indentation for this file.
2 spaces for .rb files
2 spaces for .scss files
4 spaces for .html.erb files